docs/docs/guide-tools/launch-files.md

```diff
@@ -2,11 +2,90 @@
 sidebar_position: 2
 ---
 
-# Launch files
-Basis launch files are yaml files consisting of one or two documents (separated by `---`). If there are two documents, the first is args, the second is content. If there is just one document, it's just content.
+# Launch Files
 
-## Args
+Basis launch files are YAML files used to define the launch configuration for a project. Each launch file can contain one or two documents, separated by `---`. If there are two documents, the first is for defining launch arguments (`args`), and the second is the main content of the launch (`content`). If there is only one document, it represents the launch content.
+
+### Structure of a Launch File
+
+A typical Basis launch file is organized as follows:
+
+```yaml
+# Launch arguments declaration
+args:
+  ARG_A:
+    type: <TYPE>                 # Data type of the argument (e.g., string, int)
+    default: <DEFAULT>           # Default value for the argument (optional)
+    optional: <true/false>       # Whether the argument is optional
+    help: "<HELP_STRING>"        # Optional description
+
+  ARG_B:
+    ...
+
+---
```

Comment:
inja templating only allowed below this marker - it would be nice if there was a small example as well, somehow

Reply:
added a comment and an example (further down)